mapserver.h

```diff
@@ -1517,6 +1537,11 @@ The :ref:`REFERENCE <reference>` object
   /*                              legendObj                               */
   /************************************************************************/
 
+  #define MS_LEGEND_KEYSIZE_MIN 5
+  #define MS_LEGEND_KEYSIZE_MAX 200
```

Comment:
I'm currently migrating from MS 7.6 to 8.0 and I have a problem with this limitation: So far I could provide legends consisting of a PNG like this:

```
LEGEND
   KEYSIZE 216 216
   KEYSPACING 7 7
   STATUS ON
END

CLASS
  KEYIMAGE "./inspire_lbgr/symbol/legende-sorption_matrix.png"
  NAME " "
END
```
Is there anything against raising the limit to 1000?